### PR TITLE
German data added

### PR DESCRIPTION
**Pull request by Justman10000**
This change is [Reviewable](https://reviewable.io/reviews/victornpb/deletediscordmessages/261)

### CODE REVIEW

File: deleteDiscordMessages.user.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-// @name          Undiscord - Delete all messages in a Discord channel or DM (Bulk deletion)
-// @description   Extends the discord interface so you can mass delete messages from discord
-// @namespace     https://github.com/victornpb/deleteDiscordMessages
-// @version       4.2
-// @match         https://discord.com/*
-// @downloadURL   https://raw.githubusercontent.com/victornpb/deleteDiscordMessages/master/deleteDiscordMessages.user.js
-// @homepageURL   https://github.com/victornpb/deleteDiscordMessages
-// @supportURL    https://github.com/victornpb/deleteDiscordMessages/issues
+// @name            Undiscord - Delete all messages in a Discord channel or DM (Bulk deletion)
+// @name:de         Undiscoord - Lösche alle Nachrichten in einem Discord-Kanal oder DM (Bulk-Deletion)
```
Comment (Toby222) [inline deleteDiscordMessages.user.js:3]: You made a typo
`Undisco`**`o`**`rd`

**Comment by StayWithMeSenpai** [comment]
"Bulk-Deletion" isnt really translated and i think you could replace "DM" with something more german like "Privatnachrichten"

**Comment by Justman10000** [comment]
> "Bulk-Deletion" isnt really translated and i think you could replace "DM" with something more german like "Privatnachrichten"

Fixxed

**Review by StayWithMeSenpai — APPROVED** [review]

**Comment by victornpb** [comment]
Isn't this a bit deceptive to offer a description in german but the whole application is in english?
Unfortunately I don't have the resources to maintain multiple languages in this project. And then I have no way of supporting issues and discussions in languages other than english. I can barely reserve the time to, read, tag, reply, and work on issues already...

**Pull request closed without merging**